packages/bierzo-wallet/src/routes/index.tsx

```diff
@@ -36,8 +36,17 @@ const Routes = (): JSX.Element => (
         <Route exact path={ADDRESSES_ROUTE} component={Addresses} />
         <Route exact path={TRANSACTIONS_ROUTE} component={Transactions} />
         <Route exact path={BALANCE_ROUTE} component={Balance} />
-        <Route exact path={REGISTER_PERSONALIZED_ADDRESS_ROUTE} component={RegisterUsername} />
-        <Route exact path={REGISTER_STARNAME} component={RegisterStarname} />
+        <Route
+          exact
+          path={REGISTER_IOVNAME_ROUTE}
+          component={() => <Register entity={ToRegister.Iovname} />}
```

Comment:
Please use `render` instead of `component`.
* https://reacttraining.com/react-router/core/api/Route/render-func
* https://medium.com/@migcoder/difference-between-render-and-component-prop-on-react-router-v4-368ca7fedbec